From 3a4f9e972d9723594d020b2785e3ec3ac148d02e Mon Sep 17 00:00:00 2001
From: Myy <myy@miouyouyou.fr>
Date: Tue, 17 Jan 2017 19:55:25 +0000
Subject: [PATCH 12/12] Export rockchip_pmu_set_idle_request for out-of-tree
 VPU code

In order to compile the Rockchip VPU modules "out-of-tree", it is
necessary to export rockchip_pmu_set_idle_request.

I do not know why the "out-of-tree" VPU code needs it, though. There
might be way to avoid calling this function, and avoid exporting this
function.

Or, in reverse, it might be extremely useful for external modules to
be able to use the Rockchip Power Management interface directly and,
therefore, export all the functions in rockchip/pw_domains.c

Signed-off-by: Myy <myy@miouyouyou.fr>
---
 drivers/soc/rockchip/pm_domains.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/rockchip/pm_domains.c b/drivers/soc/rockchip/pm_domains.c
index 796c46a..63a186b 100644
--- a/drivers/soc/rockchip/pm_domains.c
+++ b/drivers/soc/rockchip/pm_domains.c
@@ -178,6 +178,7 @@ static int rockchip_pmu_set_idle_request(struct rockchip_pm_domain *pd,
 
 	return 0;
 }
+EXPORT_SYMBOL(rockchip_pmu_set_idle_request);
 
 static int rockchip_pmu_save_qos(struct rockchip_pm_domain *pd)
 {
-- 
2.10.2

